utils/docker.go

```diff
@@ -284,7 +288,19 @@ func RemoveNetwork(network types.NetworkResource) {
 	}
 }
 
-// GetPort will return the current port that PFE is running on (hardcoded to 9090 for now)
-func GetPort() string {
-	return "9090";
+// GetPFEPort will return the current port that PFE is running on
+func GetPFEPort() string {
+	if CheckContainerStatus() {
+		containerList := GetContainerList()
+		for _, container := range containerList {
+			if strings.HasPrefix(container.Image, "codewind-pfe") {
```

Comment:
We have codewind-pfe hardcoded in a lot of locations now, I'm not sure that's in scope for the PR though.

Reply:
Agreed, out of scope